fuseprivate.c

```diff
@@ -148,3 +150,34 @@ int sqfs_statfs(sqfs *sq, struct statvfs *st) {
 
 	return 0;
 }
+
+void notify_mount_ready(const char *notify_pipe, int status) {
```

Comment:
Maybe instead of an int status, we can pass a constant char of 's' or 'f'? Feels a bit cleaner.